pkgs/applications/radio/gnuradio/default.nix

```diff
@@ -299,6 +299,7 @@ stdenv.mkDerivation {
   patches = [
     # Not accepted upstream, see https://github.com/gnuradio/gnuradio/pull/5227
     ./modtool-newmod-permissions.patch
+    ./fmt-10.1-compat.patch
```

Comment:
Could you comment about this? In case you write it yourself, you should know upstream is very responsive.

Reply:
Haha, I actually fixed this upstream already: https://github.com/gnuradio/gnuradio/pull/6808

I haven't checked if the patch on main applies cleanly, will switch if it does.